tommath_class.h

```diff
@@ -1,15 +1,3 @@
-/* LibTomMath, multiple-precision integer library -- Tom St Denis
- *
- * LibTomMath is a library that provides multiple-precision
- * integer arithmetic as well as number theoretic functionality.
- *
- * The library was designed directly after the MPI library by
- * Michael Fromberger but has been written from scratch with
- * additional optimizations in place.
- *
- * The library is free for all purposes without any express
- * guarantee it works.
- */
```

Comment:
missing "standard" initial comment causes `./helper.pl -a` to fail

Reply:
Sorry, @karel-m my bad, had the wrong base initially. Should be the correct one now.